src/electron.ts

```diff
@@ -21,12 +21,15 @@ function createWindow(): Electron.BrowserWindow {
     x: getPosition('width'),
     y: getPosition('height'),
     frame: false,
-    transparent: process.platform === 'win32' ? false : true
+    transparent: process.platform === 'win32' ? false : true,
+    show: false
```

Comment:
previous behaviour was that there was a blank white window shown for a second or two, until the contents loaded, now the blank window is hidden until the content is parsed and displayed